kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/collections/nulls.kt

```diff
@@ -401,30 +267,7 @@ fun <T, I : Iterable<T>> I.shouldNotContainNoNulls(): I {
  *
  * @return the same array for further assertions
  */
-fun <T> Array<T>.shouldNotContainNoNulls(): Array<T> {
-   asList().shouldNotContainNoNulls()
-   return this
-}
-
-/**
- * Asserts that this collection does not contain exclusively non-null elements.
- *
- * Verifies that the collection contains at least one null element.
- *
- * Opposite of [shouldContainNoNulls].
- *
- * Example:
- * ```
- * listOf("a", null).shouldNotContainNoNulls() // Assertion passes
- * listOf("a", "b").shouldNotContainNoNulls()  // Assertion fails
- * ```
- *
- * @return the same collection for further assertions
- */
```

Comment:
As we are keeping the public functions as is, and only removing duplicate implementations, maybe we should keep the javadoc examples?